scroll-kit.coffee

```diff
@@ -330,6 +339,9 @@ initialize_sticky = (node) ->
   node.isFixed = data.fixed or (el.css('position') is 'fixed')
   node.placeholder = placeholder(node) if update_sticky(node)
 
+  if node.data.fit
+    node.el.on('DOMMouseScroll mousewheel', preventScroll)
```

Comment:
Makes sense, did you considered using `pointer-events` for this?

I remember you can pass or deny those pointer events, such as scroll, using css only...